src/pymordemos/elliptic2.py

```diff
@@ -75,16 +82,32 @@ def elliptic2_demo(args):
         name='2DProblem'
     )
 
+    if args['PRODUCT-COUNT'] and not args['--fv']:
+        # use a random parameter to construct an energy product
+        mu_bar = problem.parameter_space.sample_randomly(1)[0]
+    else:
+        mu_bar = None
+
     print('Discretize ...')
     discretizer = discretize_stationary_fv if args['--fv'] else discretize_stationary_cg
-    m, data = discretizer(problem, diameter=1. / args['N'])
+    if mu_bar is not None:
+        m, data = discretizer(problem, diameter=1. / args['N'], mu_energy_product=mu_bar)
```

Comment:
You can use the same call, even when `mu_bar` is `None`.

Reply:
No I can't. Note that --fv does not have this argument. I changed it differently though